src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallDtoMapper cinemaHallDtoMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallDtoMapper cinemaHallDtoMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallDtoMapper = cinemaHallDtoMapper;
+    }
+
+    @PostMapping
+    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto cinemaHall) {
+        return cinemaHallDtoMapper.parse(cinemaHallService
+                .add(cinemaHallDtoMapper.toModel(cinemaHall)));
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
```

Comment:
```suggestion
    public List<CinemaHallResponseDto> getAll() {
```
as described in common mistakes, let's make our controllers' method names more consistent. Fix the same issue in all places